x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/trial_license_complete_tier/import_connectors.ts

```diff
@@ -0,0 +1,516 @@
+          action_connectors_success_count: 1,
+          action_connectors_errors: [],
+          action_connectors_warnings: [],
+        });
+
+        expect(await getConnector(supertest, CONNECTOR_ID)).toMatchObject({
+          id: CONNECTOR_ID,
+          name: 'test-connector',
+        });
+      });
+
+      it('DOES NOT import an action connector without rules', async () => {
+        const ndjson = combineToNdJson(CUSTOM_ACTION_CONNECTOR);
+
+        const { body } = await supertest
+          .post(`${DETECTION_ENGINE_RULES_URL}/_import`)
+          .set('kbn-xsrf', 'true')
+          .set('elastic-api-version', '2023-10-31')
+          .attach('file', Buffer.from(ndjson), 'rules.ndjson')
+          .expect(200);
+
+        expect(body).toMatchObject({
+          errors: [],
+          success: true,
+          success_count: 0,
+          rules_count: 0,
+          action_connectors_success: true,
+          action_connectors_success_count: 0,
+          action_connectors_errors: [],
+          action_connectors_warnings: [],
+        });
+
+        await supertest
+          .get(`/api/actions/connector/${CONNECTOR_ID}`)
+          .set('kbn-xsrf', 'foo')
+          .expect(404);
+      });
+
+      it('DOES NOT import an action connector when there are no rules referencing it', async () => {
+        const ndjson = combineToNdJson(
+          getCustomQueryRuleParams({
+            rule_id: 'rule-1',
+            name: 'Rule 1',
+            actions: [
+              {
+                group: 'default',
+                id: ANOTHER_CONNECTOR_ID,
+                params: {
+                  message: 'Some message',
+                  to: ['test@test.com'],
+                  subject: 'Test',
+                },
+                action_type_id: '.email',
+                uuid: 'fda6721b-d3a4-4d2c-ad0c-18893759e096',
+                frequency: { summary: true, notifyWhen: 'onActiveAlert', throttle: null },
+              },
+            ],
+          }),
+          { ...CUSTOM_ACTION_CONNECTOR, id: ANOTHER_CONNECTOR_ID },
+          CUSTOM_ACTION_CONNECTOR
+        );
+
+        const { body } = await supertest
+          .post(`${DETECTION_ENGINE_RULES_URL}/_import`)
+          .set('kbn-xsrf', 'true')
+          .set('elastic-api-version', '2023-10-31')
+          .attach('file', Buffer.from(ndjson), 'rules.ndjson')
+          .expect(200);
+
+        expect(body).toMatchObject({
+          errors: [],
+          success: true,
+          success_count: 1,
+          rules_count: 1,
+          action_connectors_success: true,
+          action_connectors_success_count: 1,
+          action_connectors_errors: [],
+          action_connectors_warnings: [],
+        });
+
+        await supertest
+          .get(`/api/actions/connector/${CONNECTOR_ID}`)
+          .set('kbn-xsrf', 'foo')
+          .expect(404);
+      });
+
+      it('DOES NOT return an error when rule actions reference a preconfigured connector', async () => {
```

Comment:
What in this payload indicates that the actions is referencing a `preconfigured` connector? Only the absence of the `id` in the action object?

(In this case, is this rule actually pointing to an actual preconfigured connector, or not really?)

Can you maybe add some comment to explain this?

Reply:
Ah, sorry. Just understood that the id matches what is defined in `x-pack/test/security_solution_api_integration/config/ess/config.base.ts`

Maybe add a comment pointing to that file?